.github/workflows/build_lint.yml

```diff
@@ -38,14 +49,14 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
-    - uses: actions/cache@v4
-      with:
-        path: ~/.cache/pip
-        key: ${{ runner.os }}-pip-${{ matrix.python-version }}-${{ hashFiles('Pipfile.lock') }}
-    - name: Install dependencies
-      run: pip install wheel && pip install .[dev]
+        cache: 'pip'
+    - name: Install uv
+      run: pip install uv
+    - name: Install Mesa-Geo
+      # See https://github.com/astral-sh/uv/issues/1945
+      run: uv pip install --system .[dev]
     - name: Test with pytest
-      run: pytest --cov=mesa_geo tests/ --cov-report=xml
+      run: pytest --durations=10 --cov=mesa-geo tests/ --cov-report=xml
```

Comment:
Needs to be `mesa_geo` for the package name within the code.

Reply:
Nice catch